packages/squiggle-lang/src/rescript/Reducer/Reducer_Dispatch/Reducer_Dispatch_BuiltIn.res

```diff
@@ -14,8 +14,8 @@ exception TestRescriptException
 let callInternal = (call: functionCall): result<'b, errorValue> => {
   let callMathJs = (call: functionCall): result<'b, errorValue> =>
     switch call {
-    | ("jsraise", [msg]) => Js.Exn.raiseError(toString(msg)) // For Tests
-    | ("resraise", _) => raise(TestRescriptException) // For Tests
+    | ("javascriptraise", [msg]) => Js.Exn.raiseError(toString(msg)) // For Tests
+    | ("rescriptraise", _) => raise(TestRescriptException) // For Tests
```

Comment:
why do we discard the message for rescript but not for javascript?

Reply:
also, isn't it awkward to have the word `Test` in variables of nontesty code?

Reply:
I am inserting keywords in the squiggle scripts in unit tests to see how well am I coping with unexpected exceptions.  Examples:
```
    x=1;  javascriptraise; x+1
    x=rescriptraise
```
Actually, I could device a way to move them to test modules but that would cost me a few days. Instead you have the variable implementation now. Moving this to my issue list to do after finishing the major features

Reply:
umuro/squiggle#48